### DISCUSSION TITLE
Merging courses field work + transfer + wait

### DISCUSSION THREAD

**Discussion by 555eatshit** [Courseplay/Courseplay_FS22#3262] [Q&A]
Hi there!
I'm wondering how I combine/merge courses to be able to switch between field work and transfer.
Actually I'm using CP in FS19, but discussion is closed, so I ask here...
I want to achieve that my seeder and my harvester drive to a parking position after fieldwork, but with folded tool (for the seeder).
Both vehicles shall wait on their parking lot for a defined timespan.
Harvester shall wait e.g. until seeder did his work and fields are ready for harvest again.

**Comment by 555eatshit** [comment]
I want to achieve an "infinite loop".
I tried now to first create a course from parking lot to field, combine the course from parking to field with harvest, add the option "Return to starting point".
But actually the harvester does not return...

**Comment by Tensuko** [comment]
That is not possible.
